docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Known Issues for WebView control for Windows Forms and WPF
+author: normesta
+description: This guide highlights known limitations with the current release of the WebView control for Windows Forms and WPF applications.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, WebView, Windows Forms, WPF, known issues, release notes
+---
+
+# Known Issues of the WebView control for Windows Forms and WPF applications
+
+This document describes issues and limitations in this release of the **WebView** control.
+
+## WebView control members
+
+The **WebView** control does not implement these methods of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol).
+
+* [NavigateToLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatetolocalstreamuri)
+
+* [NavigateWithHttpRequestMessage](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.navigatewithhttprequestmessage)
+
+* [CapturePreviewToStreamAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.capturepreviewtostreamasync)
+
+* [CaptureSelectedContentToDataPackageAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.captureselectedcontenttodatapackageasync)
+
+* [BuildLocalStreamUri](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.buildlocalstreamuri)
+
+The **WebView** control implements these events of the [IWebViewControl](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol), but they don't work as expected.
+
+* [LongRunningScriptDetected](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.longrunningscriptdetected)
+
+* [UnsafeContentWarningDisplaying](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsafecontentwarningdisplaying)
+
+* [UnsupportedUriSchemeIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsupportedurischemeidentified)
+
+* [UnviewableContentIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unviewablecontentidentified)
+
+* [NewWindowRequested](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.newwindowrequested)
+
+## Rendering and layout
+
+* The **WebView** control doesn't appear at the correct scale when users move the host application between monitors that have different screen resolutions.
```

Comment:
As long as they update the manifest and in WinForms case their app.config, this will work. It's probably worth putting the workaround they need to do in here.

**WinForms**
- Add the following to `app.config`
```xml
<configuration>
    <System.Windows.Forms.ApplicationConfigurationSection>
        <add key="DpiAwareness" value="PerMonitorV2"/>
    </System.Windows.Forms.ApplicationConfigurationSection>
</configuration>
```
- Add the following to app.manifest

```xml
<compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
    <application>
      <!-- Windows 10 -->
      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
    </application>
  </compatibility>
```
  - Indicate the application is DPI-aware. WPF applications are automatically DPI aware; however, this ensures we are PerMonitor DPI aware
```xml
  <application xmlns="urn:schemas-microsoft-com:asm.v3">
    <windowsSettings>
      <!-- The combination of below two tags have the following effect :
      1) Per-Monitor for >= Windows 10 Anniversary Update
      2) System < Windows 10 Anniversary Update -->
      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/PM</dpiAware>
    </windowsSettings>
  </application>
```


**WPF**

- Make the following changes to `app.manifest`
  - Ensure the application has the Windows 10 compatability
```xml
<compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
    <application>
      <!-- Windows 10 -->
      <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />
    </application>
  </compatibility>
```
  - Indicate the application is DPI-aware. WPF applications are automatically DPI aware; however, this ensures we are PerMonitor DPI aware
```xml
  <application xmlns="urn:schemas-microsoft-com:asm.v3">
    <windowsSettings>
      <!-- The combination of below two tags have the following effect :
      1) Per-Monitor for >= Windows 10 Anniversary Update
      2) System < Windows 10 Anniversary Update -->
      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitor</dpiAwareness>
      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/PM</dpiAware>
    </windowsSettings>
  </application>
```

Reply:
For the Windows Forms steps, wouldn't we just point them to the existing doc for that? - [Configuring your Windows Forms app for high DPI support](https://docs.microsoft.com/dotnet/framework/winforms/high-dpi-support-in-windows-forms#configuring-your-windows-forms-app-for-high-dpi-support).? That's what we're doing in the main WebView guide. Why list the specific steps here and not back in the main WebView guide?

Reply:
I think pointing them to the document is fine (even preferable). What about the WPF stuff? They can do the steps outlined there and it should work, though one of the switches is for WinForms only (so would have no effect on a WPF application). I'm all for not repeating ourselves